pkg/controllers/management/node/controller.go

```diff
@@ -254,11 +255,30 @@ func (m *Lifecycle) deployAgent(nodeDir string, obj *v3.Node) error {
 	if err != nil {
 		return err
 	}
+	prefix := "/"
+	cluster, err := m.clusterLister.Get("", obj.Namespace)
+	if err != nil && !kerror.IsNotFound(err) {
+		return err
+	}
+	if cluster.Spec.RancherKubernetesEngineConfig.PrefixPath != "" {
```

Comment:
Nit: can we put prefix initialization close to the place where it's used - line 277?

Reply:
Line 258-275 is to get two thing(cluster.prefix and info.osType) so the order shouldn't matter :)